technical-guidelines.md

```diff
@@ -25,19 +25,11 @@ to add the sign-off with the following command, which can then be force pushed.
 ```console
 git commit --amend -s
 ```
-
-\* Branch protection rules should protect the primary branch (usually `main`) by requiring code review from the appropriate parties (other than the author), usually expressed in a CODEOWNERS file.
-
-\*\* We recommend Renovate over Dependabot because of it's group and auto-merge features.
-Additionally, we have an org-wide base config for Renovate.
-
-\*\*\* Release Please isn't strictly necessary, but combined with [Conventional Commits][conventional-commits], it provides a simple yet robust means of generating useful, accurate changelogs and releasing semantically versioned artifacts.
-
```

Comment:
Moved this under the table so that the astricts more easily mapped to the content.